lib/components/search_user/interactor/mappers/search_user_state_mapper.dart

```diff
@@ -4,16 +4,31 @@ import 'package:seeds/domain-shared/page_state.dart';
 import 'package:seeds/domain-shared/result_to_state_mapper.dart';
 
 class SearchUserStateMapper extends StateMapper {
-  SearchUserState mapResultToState(SearchUserState currentState, Result result, List<String>? noShowUsers) {
-    if (result.isError) {
+  SearchUserState mapResultToState(
+      SearchUserState currentState, Result seedsMembersResult, Result telosResult, List<String>? noShowUsers) {
+    if (seedsMembersResult.isError && telosResult.isError) {
       return currentState.copyWith(pageState: PageState.failure, errorMessage: 'Error Searching for User');
     } else {
-      final List<MemberModel> users = result.asValue?.value as List<MemberModel>;
-      if (noShowUsers != null) {
-        for (final noShowUser in noShowUsers) {
-          users.removeWhere((element) => element.account == noShowUser);
+      List<MemberModel> seedsUsers = [];
+      List<MemberModel> telosUsers = [];
+
+      if (seedsMembersResult.isValue) {
+        seedsUsers = seedsMembersResult.asValue?.value as List<MemberModel>;
+        if (noShowUsers != null) {
+          for (final noShowUser in noShowUsers) {
+            seedsUsers.removeWhere((element) => element.account == noShowUser);
+          }
         }
       }
+
+      if (telosResult.isValue) {
+        final seedsSet = Set.from(seedsUsers.map((e) => e.account));
```

Comment:
if a telos account already appears up there in the seeds accounts - since all seeds accounts are also telos accounts - then we filter it out.